exchange/targeting.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
-const maxKeyLength = 20
+const MaxKeyLength = 20
```

Comment:
If you want, you can make this a config option rather than being hardcoded. There has been some talk about making this configurable, but no one has gotten around to it yet. Not required in this PR.

Reply:
@hhhjort I agree this should be a config option. Can we leave it for later, because this PR has first priority now?

Reply:
We already have a story for this internally. Not required for this PR.